config/locales/application_states.yml

```diff
@@ -62,16 +62,197 @@ en:
+      name: Send to provider
+      by: system
+      description: 5 days after submission the application is sent to providers. This is to give candidates the opportunity to edit their application.
+      emails:
+        - candidate_mailer-application_under_consideration
+
+    application_complete-withdraw:
+      name: Withdraw
+      by: candidate
+      description: Candidates can withdraw at any time.
+
+    awaiting_provider_decision-make_offer:
+      name: Provider makes offer
+      by: provider
+      description: |
+        Providers can make offers to candidates. Making an offer guarantees the candidate a place on the course that they applied to. Offers include conditions, that the candidate must meet in order to gain their place on a course.
+
+        All offers must include the following non-academic conditions:
+        - Candidates must pass a DBS & childen's barred list check
+        - Candidates must pass a fitness to work health check
+
+    awaiting_provider_decision-reject:
+      name: Provider rejects
+      by: provider
+      description: The provider rejects the candidate.
+
+    awaiting_provider_decision-withdraw:
+      name: Withdraw
+      by: candidate
+      description: |
+        A candidate makes a withdrawal decision to inform a provider that they no longer want their application to be considered. A candidate can withdraw an application at any time.
+        If the candidate is in Apply1, and the candidate makes the withdrawal within 5 working days of submitting their application, they can amend this application and apply for a different course choice once. See the amend application before offer feature.
+
+    awaiting_provider_decision-reject_by_default:
+      name: Reject by default
+      by: system
+      description: |
+        An application is rejected by default (RBD) if a provider doesn't make an offer within a certain number of working days after they have received an application.
+
+        The provider gets full working days so if the application was received in work hours, that day wouldn't count towards the number of decision days. Each application has an RBD time, which is set once the provider has received the application. Application state ""awaiting provider decision""
+        The RBD time of an application expires just before midnight on the last allowed working day. The calculation of midnight should take GMT/BST into account.
+
+        If the RBD time has passed and the provider hasn't made an offer or rejected the application, the DfE Apply system rejects the application automatically.
+
+        The number of decision days (that providers get) can change at various times in the recruitment cycle (e,g, it can be set to be shorter towards the end). It can also be changed (e.g. extended) for specific applications upon request from the provider or candidate (e.g. if the candidate is unable to interview within the allotted time), however this is in exceptional cases.
```

Comment:
```suggestion
        The number of decision days (that providers get) can change at various times in the recruitment cycle (e.g. it can be set to be shorter towards the end). It can also be changed (e.g. extended) for specific applications upon request from the provider or candidate (e.g. if the candidate is unable to interview within the allotted time), however this is in exceptional cases.
```